tests/ara/test_columns.py

```diff
@@ -16,7 +16,7 @@
     MostLikelyCategoryColumn(data_source="color"),
     MostLikelyProbabilityColumn(data_source="color"),
     FlatCompositionColumn(data_source="formula", sort_order=CompositionSortOrder.QUANTITY),
-    ComponentQuantityColumn(data_source="formula", component_name="Si"),
+    ComponentQuantityColumn(data_source="formula", component_name="Si", normalize=True),
```

Comment:
This is a pretty spare test (API only).  Are there any consequences we can test here?

Reply:
As far as the client is concerned these objects only get (de)serialized. There's no other logic associated with them, hence nothing to test. What we really need is an end-to-end test in nextgen-devkit.